packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -255,7 +269,13 @@
             <TextHint info text={textHintString} />
         {:else if [ProposalStatus.Commencing, ProposalStatus.Holding].includes($selectedProposal.status)}
             <buttons-container class="flex w-full space-x-4 mt-6">
-                <Button outline classes="w-full" onClick={handleCancelClick}>{localize('actions.cancel')}</Button>
+                <Button
+                    outline
+                    classes="w-full"
+                    onClick={onStopVotingClick}
+                    disabled={!isVotingForProposal || isTransferring}
+                    isBusy={isTransferring}>{localize('actions.stopVoting')}</Button
```

Comment:
I don't like this for two reasons:

1) isBusy should depend on the same variables as the disabled button.
2) Instead of relying on `isTransferring` it makes more sense to rely on a `pendingGovernanceTransactionId` being present for the selected account

Reply:
```suggestion
                    disabled={!isVotingForProposal || isTransferring}
                    isBusy={isVotingForProposal && isTransferring}>{localize('actions.stopVoting')}</Button
```